easybuild/easyconfigs/r/R/R-3.1.2-foss-2015a.eb

```diff
@@ -0,0 +1,389 @@
+    ('Biobase', '2.26.0', bioconductor_options),
+    ('S4Vectors', '0.4.0', bioconductor_options),
+    ('IRanges', '2.0.1', bioconductor_options),
+    ('GenomeInfoDb', '1.2.4', bioconductor_options),
+    ('AnnotationDbi', '1.28.1', bioconductor_options),
+    ('XVector', '0.6.0', bioconductor_options),
+    ('zlibbioc', '1.12.0', bioconductor_options),
+    ('Biostrings', '2.34.1', bioconductor_options),
+    ('GenomicRanges', '1.18.4', bioconductor_options),
+    ('Rsamtools', '1.18.2', bioconductor_options),
+    ('BiocParallel', '1.0.1', bioconductor_options),
+    ('GenomicAlignments', '1.2.1', bioconductor_options),
+    ('ShortRead', '1.24.0', bioconductor_options),
+    ('graph', '1.44.1', bioconductor_options),
+    ('gbm', '2.1', ext_options),
+    ('dichromat', '2.0-0', ext_options),
+    ('Formula', '1.1-2', ext_options),
+    ('acepack', '1.3-3.3', ext_options),
+    ('Hmisc', '3.14-6', ext_options),
+    ('munsell', '0.4.2', ext_options),
+    ('labeling', '0.3', ext_options),
+    ('scales', '0.2.4', ext_options),
+    ('fastcluster', '1.1.15', ext_options),
+    ('reshape2', '1.4.1', ext_options),
+    ('chron', '2.3-45', ext_options),
+    ('data.table', '1.9.4', ext_options),
+    ('gtable', '0.1.2', ext_options),
+    ('proto', '0.3-10', ext_options),
+    ('ggplot2', '1.0.0', ext_options),
+    ('igraph', '0.7.1', ext_options),
+    ('GeneNet', '1.2.11', ext_options),
+    ('ape', '3.2', ext_options),
+    ('htmltools', '0.2.6', ext_options),
+    ('RJSONIO', '1.3-0', ext_options),
+    ('caTools', '1.17.1', ext_options),
+    ('gplots', '2.16.0', ext_options),
+    ('ROCR', '1.0-5', ext_options),
+    ('httpuv', '1.3.2', ext_options),
+    ('R6', '2.0.1', ext_options),
+    ('shiny', '0.10.2.2', ext_options),
+    ('adegenet', '1.4-2', ext_options),
+    ('phylobase', '0.6.8', ext_options),
+    ('adephylo', '1.1-6', ext_options),
+    ('animation', '2.3', ext_options),
+    ('bigmemory.sri', '0.1.3', ext_options),
+    ('bigmemory', '4.4.6', ext_options),
+    ('calibrate', '1.7.2', ext_options),
+    ('clusterGeneration', '1.3.1', ext_options),
+    ('raster', '2.3-12', ext_options),
+    ('dismo', '1.0-5', ext_options),
+    ('expm', '0.99-1.1', ext_options),
+    ('extrafontdb', '1.0', ext_options),
+    ('Rttf2pt1', '1.3.2', ext_options),
+    ('extrafont', '0.17', ext_options),
+    ('fields', '7.1', ext_options),
+    ('shapefiles', '0.7', ext_options),
+    ('fossil', '0.3.7', ext_options),
+    ('geiger', '2.0.3', ext_options),
+    ('glmnet', '1.9-8', ext_options),
+    ('gmp', '0.5-12', ext_options),
+    ('labdsv', '1.6-1', ext_options),
+    ('MatrixModels', '0.3-1.1', ext_options),
+    ('stabs',  '0.5-0', ext_options),
+    ('mboost', '2.4-1', ext_options),
+    ('msm', '1.5', ext_options),
+    ('nor1mix', '1.2-0', ext_options),
+    ('np', '0.60-2', ext_options),
+    ('polynom', '1.3-8', ext_options),
+    ('partitions', '1.9-15', ext_options),
+    ('rgl', '0.95.1201', ext_options),
```

Comment:
this depends on OpenGL/Mesa? so it should be removed (not sure how many other things depend on it)?